tests/b013.py

```diff
@@ -13,7 +13,7 @@
 
 try:
     pass
-except (ValueError):
+except ValueError:
```

Comment:
Nice, this was one of my changes in black! 😆 we can use `# fmt: off` here to ignore formatting in this case